internal/pkg/cli/pipeline_init.go

```diff
@@ -154,11 +161,12 @@ func newInitPipelineOpts(vars initPipelineVars) (*initPipelineOpts, error) {
 		cfnClient:        cloudformation.New(defaultSession),
 		store:            ssmStore,
 		prompt:           prompter,
-		sel:              selector.NewSelect(prompter, ssmStore),
 		runner:           exec.NewCmd(),
 		fs:               &afero.Afero{Fs: afero.NewOsFs()},
 		wsAppName:        wsAppName,
 		pipelineLister:   deploy.NewPipelineStore(rg.New(defaultSession)),
+		codestar:         cs,
```

Comment:
tiny nit:
```suggestion
		codestar:         cs.New(defaultSession),
```

The real question is where do we use the member `codestar`? Does it need to be a member?